resources/views/tasks/edit.blade.php

```diff
@@ -395,494 +430,16 @@ class="mr-2 custom-badges pl-2 pr-2 rounded-lg">
     const userHasAccessToTask = {{ Auth::user()->can('update', $task) ? "true": "false" }};
     const userIsAdmin = {{ Auth::user()->is_administrator ? "true": "false" }};
     const userIsProcessManager = {{ Auth::user()->id === $task->process?->manager_id ? "true": "false" }};
-    var screenFields = @json($screenFields);
-    window.ProcessMaker.taskDraftsEnabled = @json($taskDraftsEnabled);
+    const userConfiguration = @json($userConfiguration);
 
-    window.ProcessMaker.breadcrumbs.taskTitle = @json($task->element_name);
     window.Processmaker.user = @json($currentUser);
   </script>
-    @foreach($manager->getScripts() as $script)
-        <script src="{{$script}}"></script>
-    @endforeach
-    <script src="{{mix('js/tasks/show.js')}}"></script>
-    <script>
-      const store = new Vuex.Store();
-      const main = new Vue({
-        mixins:addons,
-        store: store,
-        el: "#task",
-        data: {
-          //Edit data
-          fieldsToUpdate: [],
-          jsonData: "",
-          monacoLargeOptions: {
-            automaticLayout: true,
-          },
-          showJSONEditor: false,
-
-          // Reassignment
-          selected: null,
-          selectedIndex: -1,
-          usersList: [],
-          filter: "",
-          showReassignment: false,
-          task,
-          draftTask,
-          userHasAccessToTask,
-          hasErrors: false,
-          redirectInProcess: false,
-          formData: {},
-          submitting: false,
-          userIsAdmin,
-          userIsProcessManager,
-          showTree: false,
-          is_loading: false,
-          autoSaveDelay: 2000,
-          options: {
-            is_loading: false,
-          },
-          lastAutosave: "-",
-          lastAutosaveNav: "-",
-          errorAutosave: false,
-          formDataWatcherActive: true,
-          showInfo: true,
-          isPriority: false,
-          userHasInteracted: false,
-          caseTitle: "",
-          showMenu: true,
-          userConfiguration: @json($userConfiguration),
-          urlConfiguration:'users/configuration',
-          showTabs: true,
-        },
-        watch: {
-          task: {
-            deep: true,
-            handler(task, oldTask) {
-              window.ProcessMaker.breadcrumbs.taskTitle = task.element_name;
-              if (task && oldTask && task.id !== oldTask.id) {
-                history.replaceState(null, null, `/tasks/${task.id}/edit`);
-                this.setAllowReassignment();
-              }
-              if (task.draft) {
-                this.lastAutosave = moment(this.draftTask.updated_at).format("DD MMMM YYYY | HH:mm");
-                this.lastAutosaveNav = moment(this.draftTask.updated_at).format("MMM DD, YYYY / HH:mm");
-              } else {
-                this.lastAutosave = "-";
-                this.lastAutosaveNav = "-"
-              }
```

Comment:
@henryjonathanquispe the new code was added here https://github.com/ProcessMaker/processmaker/pull/7873/files